desc/optimize/_constraint_wrappers.py

```diff
@@ -11,9 +11,9 @@
     maybe_add_self_consistency,
 )
 from desc.objectives.utils import factorize_linear_constraints
-from desc.utils import Timer, get_instance
+from desc.utils import Timer, get_instance, setdefault
 
-from .utils import compute_jac_scale, f_where_x
+from .utils import f_where_x
```

Comment:
What is f_where_x?

Reply:
https://github.com/PlasmaControl/DESC/blob/16c89acc531655e225954cea1788f8f436838bae/desc/optimize/utils.py#L499